### DISCUSSION TITLE
object-oriented style design

### DISCUSSION THREAD

**Discussion by xiaotushaoxia** [qax-os/excelize#1578] [General]
// 目前典型的用法
f := excelize.NewFile()
f.SetCellStr(sheet2, fmt.Sprintf("A%d",  1), "a1")
f.SetCellStr(sheet2, fmt.Sprintf("B%d",  1), "b1")
f.SetCellStr(sheet2, fmt.Sprintf("C%d",  1), "c1")
f.SetCellStr(sheet2, fmt.Sprintf("D%d", 1), "d1")
// 为什么没有类似这样的  感觉简洁不少
s := excelize.NewSheet(f, "sheet2")
s.SetCellStr(fmt.Sprintf("A%d",  1), "a1")
s.SetCellStr(fmt.Sprintf("B%d",  1), "b1")
s.SetCellStr(fmt.Sprintf("C%d",  1), "c1")
s.SetCellStr(fmt.Sprintf("D%d", 1), "d1")

**Comment by xuri** [comment]
Thanks for your feedback. This was duplicate with issue #2.

**Reply by xiaotushaoxia** [reply]
这个issue提的很好，如果我当时看到这套api的时候我也会有这样的疑问，然后这个issue引出了一个File object。不过我这里提到的是Sheet object，理由和issue2差不多。不知道引入Sheet object会不会有什么问题。
